tensorflow/c/eager/BUILD

```diff
@@ -256,6 +256,73 @@ tf_cuda_cc_test(
     ],
 )
 
+cc_library(
+    name = "mnist_gradients_util",
```

Comment:
Could you run `buildifier` on the BUILD files in this CL. Another test is failing because of that.